pkg/logentry/stages/template_test.go

```diff
@@ -125,6 +125,21 @@ func TestTemplateStage_Process(t *testing.T) {
 				"missing":    "newval",
 			},
 		},
+		"template with multiple keys": {
+			TemplateConfig{
+				Source:   "output_msg",
+				Template: "{{ .level }} for app {{ .app | ToUpper }}",
+			},
+			map[string]interface{}{
+				"level": "WARN",
+				"app":   "loki",
+			},
+			map[string]interface{}{
+				"level":      "WARN",
+				"app":        "loki",
```

Comment:
should this be in upper case ?

Reply:
Not necessarily, I just made it upper case.

Reply:
ho I see